plugin/install/sql/create_views.sql

```diff
@@ -511,3 +511,77 @@ where c.date_in is not null
   and if(ccf.plugin_fields_cartridgeitemtypedropdowns_id in (2,3,4), ccf.printed_pages_color_field, ccf.printed_pages_color_field + ccf.printed_pages_field) > 0
 group by c.printers_id, ccf.plugin_fields_cartridgeitemtypedropdowns_id;
 
+create or replace view glpi_plugin_iservice_printer_usage_coefficients_v3 as
+with rankedcartridges as (
+    select
+        c.*,
+        c.suppliers_id_field suppliers_id,
+        row_number() over (partition by c.suppliers_id_field, c.printers_id, c.cartridgeitems_id order by c.date_out desc) as row_num
+    from glpi_plugin_iservice_cartridges c
+             join glpi_printers p on p.id = c.printers_id
+    where c.date_out is not null
+),
+     initialaverage as (
+         select
```

Comment:
Indentation is missing

```suggestion
create or replace view glpi_plugin_iservice_printer_usage_coefficients_v3 as
     with rankedcartridges as (
         select
             c.*,
             c.suppliers_id_field suppliers_id,
             row_number() over (partition by c.suppliers_id_field, c.printers_id, c.cartridgeitems_id order by c.date_out desc) 
     as row_num
         from glpi_plugin_iservice_cartridges c
                  join glpi_printers p on p.id = c.printers_id
         where c.date_out is not null
     ),
     initialaverage as (
         select
```